server/controllers/finance/debtorGroups.js

```diff
@@ -84,6 +87,54 @@ function create(req, res, next) {
 }
 
 /**
+* PUT /debtor_groups/:uuid
+*
+* @exemple
```

Comment:
Nice comments!  This won't block the PR, but the JSDoc tag name is `@example`, not `@exemple`. :).